docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -142,41 +142,57 @@ Edit **.detoxrc.json** and replace the configuration with:
   "runnerConfig": "e2e/config.json",
   "skipLegacyWorkersInjection": true,
   "apps": {
-    "android.release": {
-      "type": "android.apk",
-      "build": "cd android && ./gradlew :app:assembleRelease :app:assembleAndroidTest -DtestBuildType=release && cd ..",
-      "binaryPath": "android/app/build/outputs/apk/release/app-release.apk"
+    "ios.debug": {
+      "type": "ios.app",
+      "build": "xcodebuild -workspace ios/eastestsexample.xcworkspace -scheme eastestsexample -configuration Debug -sdk iphonesimulator -arch x86_64 -derivedDataPath ios/build",
+      "binaryPath": "ios/build/Build/Products/Debug-iphonesimulator/eastestsexample.app"
     },
```

Comment:
I'd drop the debug configurations here. The main guide doesn't say anything about debug builds.

```suggestion
```